cartographer/mapping/id_test.cc

```diff
@@ -79,6 +79,45 @@ TEST(IdTest, MapByIdTrajectoryRange) {
   EXPECT_TRUE(expected_data.empty());
 }
 
+TEST(IdTest, MapByIdTrajectoryIdRange) {
+  MapById<NodeId, int> map_by_id;
+  map_by_id.Append(7, 2);
```

Comment:
Since we reuse the same `MapById` many times now: Extract a test fixture?

Reply:
Done